code/modules/admin/verbs/striketeam.dm

```diff
@@ -37,33 +47,35 @@ GLOBAL_VAR_INIT(sent_strike_team, 0)
 			break
 
 	// Find ghosts willing to be DS
-	var/image/source = image('icons/obj/cardboard_cutout.dmi', "cutout_deathsquad")
-	var/list/commando_ghosts = pollCandidatesWithVeto(src, usr, COMMANDOS_POSSIBLE, "Join the DeathSquad?",, 21, 60 SECONDS, TRUE, GLOB.role_playtime_requirements[ROLE_DEATHSQUAD], TRUE, FALSE, source = source)
+	var/list/commando_ghosts = null
+	if(alert("Would you like to custom pick your Deathsquad?",, "Yes", "No") == "Yes")
+		var/image/source = image('icons/obj/cardboard_cutout.dmi', "cutout_deathsquad")
+		commando_ghosts = pollCandidatesWithVeto(src, usr, commando_number, "Join the DeathSquad?",, 21, 60 SECONDS, TRUE, GLOB.role_playtime_requirements[ROLE_DEATHSQUAD], TRUE, FALSE, source = source)
+	else
+		var/image/source = image('icons/obj/cardboard_cutout.dmi', "cutout_deathsquad")
+		commando_ghosts = shuffle(SSghost_spawns.poll_candidates("Join the Deathsquad?",, GLOB.responseteam_age, 60 SECONDS, TRUE, GLOB.role_playtime_requirements[ROLE_DEATHSQUAD], TRUE, FALSE, source = source))
+		if (length(commando_ghosts) > commando_number)
+			commando_ghosts.Cut(commando_number + 1) //cuts the ghost candidates down to the ammount requested
 	if(!length(commando_ghosts))
-		to_chat(usr, "<span class='userdanger'>Nobody volunteered to join the DeathSquad.</span>")
+		message_admins("[key_name_admin(usr)]'s Deathsquad had no volunteers and was cancelled.")
+		log_admin("[key_name(usr)]'s Deathsquad had no volunteers and was cancelled.")
+		to_chat(src, "<span class='userdanger'>Nobody volunteered to join the DeathSquad.</span>")
 		return
 
-	GLOB.sent_strike_team = 1
-
 	// Spawns commandos and equips them.
-	var/commando_number = COMMANDOS_POSSIBLE //for selecting a leader
-	var/is_leader = TRUE // set to FALSE after leader is spawned
-
-	for(var/obj/effect/landmark/spawner/ds/L in GLOB.landmarks_list)
+	for(var/obj/effect/landmark/spawner/ds/L in GLOB.landmarks_list) //Despite obj/effect/landmark/spawner/ds being in the exact same location and doing the exact same thing as obj/effect/landmark/spawner/ert, switching them breaks it.
 		if(!commando_number)
 			break
-
 		if(!length(commando_ghosts))
 			break
-
 		var/use_ds_borg = FALSE
 		var/mob/ghost_mob = pick(commando_ghosts)
 		commando_ghosts -= ghost_mob
 		if(!ghost_mob || !ghost_mob.key || !ghost_mob.client)
 			continue
 
-		if(!is_leader)
-			var/new_dstype = alert(ghost_mob.client, "Select Deathsquad Type.", "DS Character Generation", "Organic", "Cyborg")
+		if(is_leader)
```

Comment:
Not sure that allowing *only* the leader to be borg is a good idea.

Reply:
Fixed, changed this in tested to test the borgs